### PR TITLE
Fix system.s3queue_log

### PR DESCRIPTION
**Pull request by kssenii**
### Changelog category (leave one):
- Bug Fix (user-visible misbehavior in an official stable release)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Fix non-filled column `table_uuid` in `system.s3queue_log`. Added columns `database` and `table`. Renamed `table_uuid` to `uuid`.

### CODE REVIEW

**Comment by robot-ch-test-poll3** [comment]
*This is an automated comment for commit 3dac4c97438317db2357d0e1db1756a2638da55b with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/60166/3dac4c97438317db2357d0e1db1756a2638da55b/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/60166/3dac4c97438317db2357d0e1db1756a2638da55b/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

File: src/Interpreters/S3QueueLog.cpp [hunk 1]
```diff
@@ -28,7 +28,9 @@ ColumnsDescription S3QueueLogElement::getColumnsDescription()
         {"hostname", std::make_shared<DataTypeLowCardinality>(std::make_shared<DataTypeString>())},
         {"event_date", std::make_shared<DataTypeDate>()},
         {"event_time", std::make_shared<DataTypeDateTime>()},
-        {"table_uuid", std::make_shared<DataTypeString>()},
+        {"database", std::make_shared<DataTypeString>()},
```
Comment (vdimir) [inline src/Interpreters/S3QueueLog.cpp:31]: Will we just will rename old table to `s3queue_log_0` on upgrade?

**Review by vdimir — APPROVED** [review]

File: src/Interpreters/S3QueueLog.cpp (hunk 1 above)
Comment (kssenii) [reply src/Interpreters/S3QueueLog.cpp:31]: yes should be so

**Comment by kssenii** [comment]
Integration tests: test_replicated_database, test_replicated_table_attach
Stateless: 00984_parser_stack_overflow

**Pull request merged**